service/src/main/java/com/codeforcommunity/requester/Emailer.java

```diff
@@ -52,6 +53,11 @@ public Emailer(DSLContext db) {
     this.pfRequestUrl =
         String.format(
             "%s%s", PropertiesLoader.loadProperty("frontend_base_url"), "/family-requests");
+    this.verifyEmailUrl =
+        String.format(
+            "%s%s",
+            frontendProperties.getProperty("frontend_base_url"),
+            frontendProperties.getProperty("verify_email_route"));
```

Comment:
```suggestion
            PropertiesLoader.getProperty("frontend_base_url"),
            PropertiesLoader.getProperty("verify_email_route"));
```
I messed up when merging :/